src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,224 @@
+  Android is not supported because it requires you to show a notification,
+  though this may change in the future. Chrome OS support is pending
+  implementation of badging on the platform.
+{% endAside %}
+
+### Register for the origin trial {: #ot }
+
+1. [Request a token][ot-request] for your origin.
+2. Add the token to your pages, there are two ways to provide this token on
+   any pages in your origin:
+     * Add an `origin-trial` `<meta>` tag to the head of any page. For example,
+       this may look something like: <br>
+       `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+     * If you can configure your server, you can also provide the token on pages
+       using an `Origin-Trial` HTTP header. The resulting response header should
+       look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+### Alternatives to the origin trial
+
+If you want to experiment with the Badging API locally, without an origin trial,
+enable the `#enable-experimental-web-platform-features` flag in `chrome://flags`.
+
+### Using the Badging API during the origin trial
+
+{% Aside 'gotchas' %}
+  During the origin trial, the API will be available via
+  `window.ExperimentalBadge`. The code below is based on the current design,
+  and will change before it lands in the browser as a standardized API.
+{% endAside %}
+
+To use the Badging API, your web app needs to meet
+[Chrome's installability criteria](https://developers.google.com/web/fundamentals/app-install-banners/#criteria),
+and users must add it to their home screens.
+
+The `ExperimentalBadge` interface is a member object on `window`. It contains
+two methods:
+
+* `set([number])`: Sets the app's badge. If a value is provided, set the badge
+  to the provided value otherwise, display a plain white dot (or other flag as
+  appropriate to the platform).
+* `clear()`: Removes app's badge.
+
+For example:
+
+```js
+// In a web page
+const unreadCount = 24;
+window.ExperimentalBadge.set(unreadCount);
+```
+
+`ExperimentalBadge.set()` and `ExperimentalBadge.clear()` can be called from
+a foreground page, or potentially in the future, a service worker. In either
+case, it affects the whole app, not just the current page.
+
+In some cases, the OS may not allow the exact representation of the badge,
+in this case, the browser will attempt to provide the best representation for
+that device. For example, while the Badging API isn't supported on Android,
+Android only ever shows a dot instead of a numeric value.
+
+Don't assume anything about how the user agent wants to display the badge.
+We expect some user agents will take a number like "4000" and rewrite it as
+"99+". If you saturate the badge yourself (for example by setting it to "99")
+then the "+" won't appear. No matter the actual number, just set
+`Badge.set(unreadCount)` and let the user agent deal with displaying it
+accordingly.
+
+## Feedback {: #feedback }
+
+We want to hear about your experiences with the Badging API.
```

Comment:
This use of "we" is fine IMO because the sections below make it clear who "we" represents in each context

Reply:
I'm wondering if it should say "the Chrome team" instead of "we"? I made a similar comment on Yoav's post for responsive images where I asked him to define "we" as "The Web Performance Working Group".

I'd like, as much as possible, for us to try to be browser agnostic where we can, or clearly identify when Chrome is doing something. I think this helps with web.dev's neutrality and ensuring that other browsers feel like it's a resource they can link to and contribute to.

Reply:
Updated.